integration/tests/execution/suite.go

```diff
@@ -126,12 +126,16 @@ func (s *Suite) SetupTest() {
 		s.nodeConfigs = append(s.nodeConfigs, nodeConfig)
 	}
 
-	// need one execution nodes
+	// at least 2 execution nodes are required to execute a script with the default AgreeingExecutorsCount, which is 2
```

Comment:
for the purposes of the execution suite, if this was failing because of the `AgreeingExecutorsCount` setting, I think we should just update the AN config to set it to 1. you may need Taras's PR merged first.